ubuntu-headless-qt/script/apt_wifi_ble.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Check if the script is run as root
```

Comment:
Add script description

Reply:
This script is redundant and will be removed